client/selectors/index.js

```diff
@@ -6,6 +6,8 @@ import { ITEM_STATE } from '../constants'
 
 export const getAgendas = (state) => state.agenda.agendas
 export const getCurrentPlanningId = (state) => state.planning.currentPlanningId
+export const getCurrentSearch = (state) => state.events.search.currentSearch ?
+    state.events.search.currentSearch : {}
```

Comment:
I recently discovered it's better to avoid to return default values in selector (like `{}` here) because it repeat what the reducer's initial state declares. And when it changes somewhere, there is a risk to forget to replace the default value everywhere.